TYPO3.Neos/Documentation/ExtendingNeos/CustomBackendModules.rst

```diff
@@ -0,0 +1,94 @@
+.. _custom-backend-modules:
+
+Custom Backend Modules
+======================
+
+If you want to integrate custom tools into the neos-backend you can do so by adding a submodule to the
+administration- or the management-section of the main menu.
+
+Some possible use cases would be the integrating of mass editing options, triggering of import or export
+actions or creating of editing interfaces for DomainModels from FlowPackages.
+
+.. warning:: This is no pubic api yet and still might change in future releases of Neos.
```

Comment:
This is not public API yet due to it's unpolished state and is subjectable to change in the future